moveit_ros/moveit_servo/include/moveit_servo/servo.hpp

```diff
@@ -126,10 +127,21 @@ class Servo
 
   /**
    * \brief Smoothly halt at a commanded state when command goes stale.
-   * @param The last commanded joint states.
+   * @param halt_state_maybe A std::optional containing the last commanded joint states, if available.
```

Comment:
```suggestion
```
Outdated comment

Reply:
This seems like the type of thing that should be a free function, eh? Transparent inputs and outputs

Reply:
It is a public API of `Servo` which is used by `ServoNode`, not sure if making it a free function would make sense contextually  even though it can easily be a free function.